src/App.jsx

```diff
@@ -1,34 +1,37 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import * as React from 'react';
+
+const todoList = [
+  {
+  id: 1,
+  title: 'Complete assignment'
+},
+{
+  id: 2,
+  title: 'Attend mentor session'
+},
+{
+  id: 3,
+  title: 'Read related articles'
+}
+];
+
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <div>
+      <h1>Todo List</h1>
+
+      <ul>
```

Comment:
You can avoid return in return just using arrow function. And, because you just return html you can skip {} bracket after `=>` and just use implicit return: 
<img width="430" alt="image" src="https://github.com/nihalerdal/react-todo/assets/106414157/5283a34a-b789-4297-a03e-d95dd06c6a56">

The [MDN Web Docs link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
<img width="803" alt="image" src="https://github.com/nihalerdal/react-todo/assets/106414157/575eff49-c8dc-4537-9814-9abcd46cf487">

Reply:
Thanks, arrow func added instead of return